public/main.js

```diff
@@ -60,7 +63,15 @@ function reloadTodoList() {
                 deleteEntry(todo);
             });
 
+            var completeButton = document.createElement("button");
```

Comment:
I think this has got to the point where we have 2 buttons, so refactoring out to a function that creates buttons would be really good.